ml/constants.py

```diff
@@ -0,0 +1,21 @@
+# ############################################################
+# source and destination labels for consistent pipe lining
+SRC = "_src"
+DST = "_dst"
+WEIGHT = "_weight"
+
+# #############################
+# consistent clf pipelining
+FEATURE = "feature"
```

Comment:
should this be an array of feature cols, `NODE_FEATURE_COLS = [ ...]`, `EDGE_FEATURE_COLS = [ ... ]` ? 

and as a local prop of `PlotterBase` so folks can functionally manipulate variants?

Not sure yet what they're doing